src/OrchardCore.Cms.Web/Program.cs

```diff
@@ -8,16 +8,18 @@ namespace OrchardCore.Cms.Web
 {
     public class Program
     {
-        public static Task Main(string[] args)
-            => BuildHost(args).RunAsync();
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
 
-        public static IHost BuildHost(string[] args) =>
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
                 .ConfigureLogging(logging => logging.ClearProviders())
```

Comment:
This breaks automatic startup of the website in Visual Studio Code.

Reply:
updated to `launch.json` to use `serverReadyAction`